<!DOCTYPE html>
<!--
    Plain-Academic by Vasilios Mavroudis
    Released under the Simplified BSD License/FreeBSD (2-clause) License.
    https://github.com/mavroudisv/plain-academic
-->

<html lang="en">
<head>
  <title>Ross S McInnes</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  <link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
  <meta name="google-site-verification" content="Bk3Bgh9icZirqwjV7BszxyOl3Kxr3klMDilk2_ir5mw" />	
</head>
<body>


<!-- Navigation -->
	<nav class="navbar navbar-inverse navbar-static-top" role="navigation">
	  <div class="container">
		<div class="navbar-header">
		  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
		</div>

		<!-- Collect the nav links, forms, and other content for toggling -->
		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
		  <ul class="nav navbar-nav">
				  <li><a href="index.html">Home</a></li>
			          <li><a href="#background">Background</a></li>
			          <li><a href="#qualifications">Qualifications</a></li>
			  	  <li><a href="#research">Research Interests</a></li>
			  	  <li><a href="#expertise">Expertise</a></li>
				  <li><a href="https://scholar.google.com/citations?user=K4vdFwEAAAAJ&hl=en&oi=ao">Publications</a></li> 
				  <li><a href="./CV_Ross_McInnes_2023.pdf">CV</a></li> 
		  </ul>
		</div>
	  </div>
	</nav>
  
  <!-- Page Content -->
    <div class="container">

            <!-- Entries Column -->
            <div class="col-md-8">
                
                <!-- Main Image -->
                <img class="img-responsive" src="RSM.jpg" width="300" height="50" alt=""><br>
                <h2 id="background">Background</h2>
                <div style="margin-top:3%; text-align:justify;">                
					<p> I am currently a Postdoctoral Researcher at the Cancer Research UK Scotland Institute (formerly known as the Beatson Institute for Cancer Research) in Glasgow, Scotland. 
					My research at the CRUK SI focuses on the use of <i>Salmonella</i> Typhimurium as a bacterial cancer therapy (BCT). 
					Leveraging my previous experience in molecular microbiology and bioinformatics, I am using innovative techniques to investigate the transcriptome of tumour organoids and <i>Salmonella</i> Typhimurium during BCT. 
					As well as tumour organoids, we are currently using mouse models of colorectal cancer (CRC) to understand the host response to CRC and BCT. 
					I am also interested in the role that the gut microbiome plays in colorectal cancer and its role in modulating BCT. 
					My training is as bacteriologist with my previous work focusing on the use of different omics techniques to investigate antibiotic resistance in the gut microbiome with a particular focus on the opportunistic pathogen <i>Enterococcus faecium</i>. 
					My current position allows me to use the skills and knowledge that I have previously acquired in the gut microbiome field to develop novel and innovative cancer therapies.  </p>
					
                </div>
            </div> 

            <!-- Contact Info on the Sidebar -->
            <div class="col-md-4">
                <div style="font-family: 'Oswald', sans-serif; font-size: 32px;"><b>Dr Ross S McInnes</b></div><br>
                <p><b>r.mcinnes@crukscotlandinstitute.ac.uk</b><br>
                <p>Cancer Research UK Scotland (Beatson) Institute<br>
                Switchback Rd<br>
                Bearsden<br>
                Glasgow<br>
                UK<br>
                </p>
            </div>
            
            <!-- Links on the Sidebar -->
            <div class="col-md-4" style="margin-top:2%">
  			<div class="column">
    				<a href="https://twitter.com/roscobacterium"><img src="twitter.png" style="width:50px;height:50px;"></a>
  			</div>
  			<div class="column">
    				<a href="https://www.linkedin.com/in/ross-mcinnes-19b095a8/"><img src="linkedin-3.png" style="width:50px;height:50px;"></a>
  			</div>
            </div>

	    	<!-- Previous positions -->
		<div class="col-md-8">
			<h2>Previous positions</h2>
			<ul>
				<li> Postdoctoral Research Fellow, University of Birmingham, England, 2022-2024. </li>
			</ul>
		</div>

		<!-- Qualifications -->
		<div class="col-md-8">
			<h2 id="qualifications">Qualifications</h2>
			<ul>
				<li> PhD Antimicrobials and Antimicrobial Resistance, University of Birmingham, England, 2022. </li>
				<li> MRes Antimicrobials and Antimicrobial Resistance, University of Nottingham and University of Birmingham, England, 2018. </li>
				<li> BSc Biochemistry and Microbiology Hons (First Class), University of Strathclyde, Scotland, 2017. </li>
			</ul>
		</div>

		<!-- Research Interests -->
		<div class="col-md-8">
			<h2 id="research">Research Interests</h2>
			<h4><i>Enterococcus faecium</i></h4>
			<p align="justify"><i>Enterococcus faecium</i> is a low-GC Gram-positive opportunistic pathogen that is associated with the hospital environment. This organism is a major cause of urinary tract infections, endocarditis, and bloodstream infections in humans. It is a particularly hardy organism that can survive in nutrient limited conditions and is resistant to many clinically important antibiotics. <i>E. faecium</i> strains are increasingly resistant to the glycopeptide antibiotic vancomycin. Resistance to vancomycin is conferred by a number of well characterised vancomycin resistance genes of which <i>vanA</i> and <i>vanB</i> are the most common. The vancomycin resistance genes are found on mobile genetic elements including conjugative plasmids and integrative and conjugative elements which means that they can be readily transferred between strains.</p>
			<p>My current interests in <i>Enterococcus faecium</i> include:</p>
			<ul>
				<li> Vancomycin resistant <i>E. faecium</i> (VRE) hospital outbreaks. </li>
				<li> Alternative mechanisms of vancomycin tolerance. </li>
				<li> Vancomycin-variable <i>E. faecium</i> strains. </li>
				<li> Simultaneous carriage of different sequence types. </li>
				<li> Horizontal transfer of vancomycin resistance genes. </li>
			</ul>
			
			<img class="img-responsive" src="enterococcus.jpg" alt=""><br>
			
			<h4> Human gut microbiota </h4>
			<p align="justify"> The gut microbiota is the community of microorganisms (bacteria, fungi, viruses etc) that live within the human gastrointestinal tract. It has many important functions including colonisation resistance, nutrient metabolism, and immune stimulation. The gut microbiota contains opportunistic pathogens that are normally kept at low levels by the other gut commensals. However, in cases of immunodeficiency or perturbation by antibiotics, these opportunistic pathogens can “bloom” and become the major component of the gut. The increased abundance of these organisms makes it more likely that they can cross into the bloodstream and cause disease. It is also hypothesised that the commensal bacteria within the gut microbiota can act as a reservoir for antibiotic resistance gene which can be transferred to the opportunistic pathogens.</p>
			<p> My current interests in the human gut microbiota include:</p>
			<ul>
				<li> Horizontal gene transfer of antibiotic resistance genes within the gut. </li>
				<li> Linking antibiotic resistance genes to their bacterial hosts. </li>
				<li> The contribution of the gut microbiota to human disease. </li>
			</ul>

			<img class="img-responsive" src="meta.jpg" alt=""><br>
		</div>

		<!-- Expertise -->
		<div class="col-md-8">
			<h2 id="expertise">Expertise</h2>
			<li> Genomics. </li>
			<li> Shotgun metagenomic sequencing. </li>
			<li> Transposon insertion sequencing. </li>
			<li> RNA sequencing. </li>
			<li> Long-read sequencing. </li>
			<li> Genetic manipulation of Gram-positive organisms. </li>
		</div>
		
		<!-- Publications -->
		<div class="col-md-8">
			<h2 id="publications">Selected Publications</h2>
			<p><b>McInnes, R.S.</b>, Snaith, A.E., Dunn, S.J., Papangeli, M., Hardy, K.J., Hussain, A., Van Schaik, W. (2024) ‘Integration of <i>vanHAX</i> downstream of a ribosomal RNA operon restores vancomycin resistance in a susceptible <i>Enterococcus faecium</i> strain’, <i>npj Antimicrob Resist</i>, 2, 2; doi:10.1038/s44259-023-00017-0.</p>
			<p><b>McInnes, R.S.</b>, Uz-Zaman, M.H., Alam, I.T., Ho, S.F.S., Moran, R.A., Clemens, J.D., Islam, M.S., Van Schaik, W. (2021) ‘Metagenome-Wide Analysis of Rural and Urban Surface Waters and Sediments in Bangladesh Identifies Human Waste as a Driver of Antibiotic Resistance’, <i>mSystems</i>, 6(4), pp. e00137-21. doi:10.1128/mSystems.00137-21.</p>
			<p><b>McInnes, R.S.</b>, McCallum, G.E., Lamberte, L.E., Van Schaik, W. (2020) ‘Horizontal transfer of antibiotic resistance genes in the human gut microbiome’, <i>Current Opinion in Microbiology</i>, 53, pp. 35–43. doi:10.1016/j.mib.2020.02.002.</p>
		</div>
	</div>
    </div>
    <!-- /.container -->
    
    <!-- Other people may like it too! -->
    <a style="color:#b5bec9;font-size:0.8em; float:right;" href="https://github.com/mavroudisv/plain-academic">Plain Academic</a> 
    
</body>

</html>
