<!DOCTYPE html>
<!--
    Plain-Academic by Vasilios Mavroudis
    Released under the Simplified BSD License/FreeBSD (2-clause) License.
    https://github.com/mavroudisv/plain-academic
-->

<html lang="en">
<head>
  <title>Ross S McInnes</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  <link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
</head>
<body>


<!-- Navigation -->
	<nav class="navbar navbar-inverse navbar-static-top" role="navigation">
	  <div class="container">
		<div class="navbar-header">
		  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
		</div>

		<!-- Collect the nav links, forms, and other content for toggling -->
		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
		  <ul class="nav navbar-nav">
				  <li><a href="index.html">Home</a></li>
			          <li><a href="#background">Background</a></li>
			          <li><a href="#qualifications">Qualifications</a></li>
			  	  <li><a href="#research">Research Interests</a></li>
			  	  <li><a href="#expertise">Expertise</a></li>
				  <li><a href="https://scholar.google.com/citations?user=K4vdFwEAAAAJ&hl=en&oi=ao">Publications</a></li> 
				  <li><a href="./CV_Ross_McInnes_2023.pdf">CV</a></li> 
		  </ul>
		</div>
	  </div>
	</nav>
  
  <!-- Page Content -->
    <div class="container">

        <div class="row">

            <!-- Entries Column -->
            <div class="col-md-8">
                
                <!-- Main Image -->
                <img class="img-responsive" src="RSM.jpg" width="300" height="50" alt=""><br>
                <h2 id="background">Background</h2>
                <div style="margin-top:3%; text-align:justify;">                
					<p> I am currently a Postdoctoral Research Fellow within the Institute of Microbiology and Infection (IMI) at the University of Birmingham, UK.
					My interests lie mainly with antibiotic resistance and the gut microbiome. During my PhD I worked with the opportunistic pathogen <i>Enterococcus 
					faecium</i>. I uncovered genes that contribute to increased fitness in the presence of the glycopeptide antibiotic vancomycin and identified 
					a novel mechanism by which a supposedly vancomycin susceptible <i>Enterococcus faecium</i> strain could revert to a resistant phenotype. My current
					JPIAMR funded project aims to investigate antibiotic resistance and horizontal gene transfer in the human gut microbiome. In this project we are using
					the novel technique Hi-C to link antibiotic resistance genes to their bacterial hosts. Nanopore long-read sequencing and anaerobic culturing are also
					being used to investigate the genetic context of antibiotic resistance genes. </p>
					
                </div>
            </div> 

            <!-- Contact Info on the Sidebar -->
            <div class="col-md-4">
                <div style="font-family: 'Oswald', sans-serif; font-size: 32px;"><b>Dr Ross S McInnes</b></div><br>
                <p><b>r.mcinnes@bham.ac.uk</b><br>
                <p>Institute of Microbiology and Infection<br>
                University of Birmingham<br>
                Edgbaston<br>
                Birmingham <br>
                UK<br>
                </p>
            </div>
            
            
            <!-- Links on the Sidebar -->
            <div class="col-md-4" style="margin-top:2%">
              <dd><a href="https://twitter.com/roscobacterium">Twitter</a></dd>
              <dd><a href="https://www.linkedin.com/in/ross-mcinnes-19b095a8/">LinkedIn</a></dd>
            </div>

		<!-- Qualifications -->
		<div class="col-md-8">
			<h2 id="qualifications">Qualifications</h2>
			<ul>
				<li> PhD Antimicrobials and Antimicrobial Resistance, University of Birmingham, England, 2022. </li>
				<li> MRes Antimicrobials and Antimicrobial Resistance, University of Nottingham and University of Birmingham, England, 2018. </li>
				<li> BSc Biochemistry and Microbiology Hons (First Class), University of Strathclyde, Scotland, 2017. </li>
			</ul>
		</div>

		<!-- Research Interests -->
		<div class="col-md-8">
			<h2 id="research">Research Interests</h2>
			<h3><i>Enterococcus faecium</i></h3>
			<p><i>Enterococcus faecium</i> is a low-GC Gram-positive opportunistic pathogen that is associated with the hospital environment. This organism is a major cause of urinary tract infections, endocarditis, and bloodstream infections in humans. It is a particularly hardy organism that can survive in nutrient limited conditions and is resistant to many clinically important antibiotics. <i>E. faecium</i> strains are increasingly resistant to the glycopeptide antibiotic vancomycin. Resistance to vancomycin is conferred by a number of well characterised vancomycin resistance genes of which <i>vanA</i> and <i>vanB</i> are the most common. The vancomycin resistance genes are found on mobile genetic elements including conjugative plasmids and integrative and conjugative elements which means that they can be readily transferred between strains.</p>
			<p>My current interests in Enterococcus faecium include:</p>
			<ul>
				<li> Vancomycin resistant <i>E. faecium</i> (VRE) hospital outbreaks. </li>
				<li> Alternative mechanisms of vancomycin tolerance. </li>
				<li> Vancomycin-variable <i>E. faecium</i> strains. </li>
				<li> Simultaneous carriage of different sequence types. </li>
				<li> Horizontal transfer of vancomycin resistance genes. </li>
			</ul>
			<h3> Human gut microbiota </h3>
			<p> The gut microbiota is the community of microorganisms (bacteria, fungi, viruses etc) that live within the human gastrointestinal tract. It has many important functions including colonisation resistance, nutrient metabolism, and immune stimulation. The gut microbiota contains opportunistic pathogens that are normally kept at low levels by the other gut commensals. However, in cases of immunodeficiency or perturbation by antibiotics, these opportunistic pathogens can “bloom” and become the major component of the gut. The increased abundance of these organisms makes it more likely that they can cross into the bloodstream and cause disease. It is also hypothesised that the commensal bacteria within the gut microbiota can act as a reservoir for antibiotic resistance gene which can be transferred to the opportunistic pathogens.</p>
			<p> My current interests in the human gut microbiota include:</p>
			<ul>
				<li> Horizontal gene transfer of antibiotic resistance genes within the gut. </li>
				<li> Linking antibiotic resistance genes to their bacterial hosts. </li>
				<li> The contribution of the gut microbiota to human disease. </li>
			</ul>
		</div>

		<!-- Expertise -->
		<div class="col-md-8">
			<h2 id="expertise">Expertise</h2>
			<li> Genomics. </li>
			<li> Shotgun metagenomic sequencing. </li>
			<li> Transposon insertion sequencing. </li>
			<li> RNA sequencing. </li>
			<li> Long-read sequencing. </li>
			<li> Genetic manipulation of Gram-positive organisms. </li>
		</div>
		
		<!-- Publications -->
		<div class="col-md-8">
			<h2 id="publications">Selected Publications</h2>
			<p><b>McInnes, R.S.</b>, McCallum, G.E., Lamberte, L.E., Van Schaik, W. (2020) ‘Horizontal transfer of antibiotic resistance genes in the human gut microbiome’, <i>Current Opinion in Microbiology</i>, 53, pp. 35–43. doi:10.1016/j.mib.2020.02.002.</p>
			<p><b>McInnes, R.S.</b>, Uz-Zaman, M.H., Alam, I.T., Ho, S.F.S., Moran, R.A., Clemens, J.D., Islam, M.S., Van Schaik, W. (2021) ‘Metagenome-Wide Analysis of Rural and Urban Surface Waters and Sediments in Bangladesh Identifies Human Waste as a Driver of Antibiotic Resistance’, <i>mSystems</i>, 6(4), pp. e00137-21. doi:10.1128/mSystems.00137-21.</p>
			<p><b>McInnes, R.S.</b>, Snaith, A.E., Dunn, S.J., Papangeli, M., Hardy, K.J., Hussain, A., Van Schaik, W. (2023) ‘A novel resistance reversion mechanism in a vancomycin-variable Enterococcus faecium strain’, <i>bioRxiv</i>, 2023.04.28.538149; doi:10.1101/2023.04.28.538149</p>
		</div>
	</div>
    </div>
    <!-- /.container -->
    
    <!-- Other people may like it too! -->
    <a style="color:#b5bec9;font-size:0.8em; float:right;" href="https://github.com/mavroudisv/plain-academic">Plain Academic</a> 
    
</body>

</html>
